### PR TITLE
docs: correct typos & improve wording

### PR DESCRIPTION
**Pull request by lnxcz**
As title says, I've made some improvements to the sentry-ui README file 🙂

### CODE REVIEW

**Comment by adrien2p** [comment]
Thanks a lot, could i get you to update the doc as well in the doc dir 🙏

File: packages/medusa-plugin-sentry-ui/README.md [hunk 1]
```diff
@@ -21,21 +21,21 @@
```
Comment (adrien2p) [inline packages/medusa-plugin-sentry-ui/README.md:21]: suggestion: Could you add a link to the documentation website to the right location please

**Review by adrien2p — CHANGES_REQUESTED** [review]

File: packages/medusa-plugin-sentry-ui/README.md (hunk 1 above)
Comment (lnxcz) [reply packages/medusa-plugin-sentry-ui/README.md:21]: Oh yea, sure

**Comment by adrien2p** [comment]
Thanks a lot 🙏

**Pull request merged**